package.json

```diff
@@ -103,6 +103,7 @@
     "pretty-ms": "^7.0.1",
     "require-relative": "^0.8.7",
     "requirejs": "^2.3.6",
+    "rewire": "^5.0.0",
```

Comment:
This may not be needed depending how we test.

Reply:
Removed